runner.go

```diff
@@ -21,43 +26,57 @@ func (r *runnerService) Run(ctx context.Context, opts RunOpts) error {
 	}
 
 	procName := opts.Command[0]
-	proc := Process{
-		Quantity: 1,
-	}
+	var proc Process
```

Comment:
same question about this method - it's getting a little long, worth breaking it up?